support/mender-inventory-network

```diff
@@ -38,7 +38,7 @@ for devpath in $SCN/*; do
         continue
     fi
     echo "mac_$dev=$(cat $devpath/address)"
-    echo "ifaces=$dev"
+    echo "network_interfaces=$dev"
```

Comment:
you need to update example output  https://github.com/mendersoftware/mender/pull/170/commits/9c8359c4f7f49511a70128c16ae601c697a281eb#diff-971ec20e2833e1ee3573c1e1baeee0a6R18